flycheck.el

```diff
@@ -1803,14 +1802,15 @@ CHECKER is a checker symbol.
 
 Pop up a help buffer with the documentation of CHECKER."
   (interactive
-   (let* ((checker (flycheck-checker-at-point))
+   (let* ((checker-at-pt (flycheck-checker-at-point))
           (enable-recursive-minibuffers t)
```

Comment:
I wonder what I added this binding for?  Do we actually use recursive mini buffers here?!